doc/middleware-it-registratore-telematico/operation-modes/on-premise-platforms/windows.md

```diff
@@ -0,0 +1,27 @@
+---
+slug: /poscreators/middleware-doc/italy/platforms/windows
+title: Windows
+---
+
+# fiskaltrust.Middleware für Windows
+
+The fiskaltrust.Middleware runs as a service on all 32-bit and 64-bit platforms starting from Windows 7. Older operating systems are not supported as they do not meet the legal requirements of a "safe system environment".
+
+| Operating system                    | Support       |
```

Comment:
Maybe mention that we follow the .net support lifecycle (we support the same OSs as the current LTS .NET versions) and link to the pages?

Reply:
https://learn.microsoft.com/en-us/lifecycle/products/microsoft-net-and-net-core